torchbenchmark/models/nvidia_deeprecommender/origin

```diff
@@ -0,0 +1 @@
+origin	https://github.com/nvidia/deeprecommender
```

Comment:
Looks like this file can be removed?

Reply:
The ADD_MODELS doc says to add an origin file with original source of test code. I made it the same as the attention_is_all_you_need_pytorch model. Is there some other format we're supposed to use?

Reply:
it looks good to me. why remove @xuzhao9?

(it's possible that some people haven't been following that suggestion, but it is something i wanted done for every model)

Reply:
> it looks good to me. why remove @xuzhao9?
> 
> (it's possible that some people haven't been following that suggestion, but it is something i wanted done for every model)

Sorry I missed the instruction in ADD_MODELS. Please ignore my comment.